packages/ui-components/Common/AlertBox/index.module.css

```diff
@@ -1,43 +1,28 @@
 .alertBox {
-  @apply flex
-    flex-row
-    items-center
-    gap-2
-    rounded
-    px-4
-    py-3
-    text-sm
-    text-white;
+  @apply flex flex-row items-center gap-2 rounded px-4 py-3 text-sm text-white;
```

Comment:
Would you be able to return to the old styling? It makes it easier to read to separate classes per line and it was intentionally done like that :P

Reply:
This applies to all .module.css files btw

Reply:
Will do! Looks like prettier might have done some formatting 😅